web/client/actions/catalog.js

```diff
@@ -15,6 +15,7 @@ const RECORD_LIST_LOADED = 'RECORD_LIST_LOADED';
 const RECORD_LIST_LOAD_ERROR = 'RECORD_LIST_LOAD_ERROR';
 const CHANGE_CATALOG_FORMAT = 'CHANGE_CATALOG_FORMAT';
 const ADD_LAYER_ERROR = 'ADD_LAYER_ERROR';
+const RESET_STATUS = 'RESET_STATUS';
```

Comment:
This is too generic. You can call the variable RESET_STATUS but the string should be unique, lets say `CATALOG_RESET`